samples/snippets/sync_api.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloudoptimization_sync_api]
+
+import google.auth
+from google.cloud import optimization_v1
+
+
+def call_sync_api():
+  """Call the sync api for fleet routing."""
+  # Use the default credentials for the environment.
+  _, project_id = google.auth.default()
+  # Change the file name to your request file.
+  request_file_name = "resources/sync_request.json"
+  fleet_routing_client = optimization_v1.FleetRoutingClient()
+
+  with open(request_file_name, 'r') as f:
+    # The request must include the `parent` field with the value set to
+    # 'projects/{YOUR_GCP_PROJECT_ID}'.
+    fleet_routing_request = optimization_v1.OptimizeToursRequest.from_json(f.read())
+    fleet_routing_request.parent = "projects/{your_project_id}".format(your_project_id = project_id)
```

Comment:
Please use f-strings unless this is for a product supporting Python 2 or Python 3.5 or below

Reply:
Also, instead of having developers change their project ID here, follow the code snippet style guide where you have a TODO block elsewhere in the code for devs to fill in their values, and then use those values in your code https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/snippets.md?cl=head

Reply:
Updated to use f-strings. Also added the TODO comment and moved project_id to function input arg for easier testing.